### PR TITLE
Remove user service

### PR DESCRIPTION
**Pull request by michaeledavis-credera**
Also removes the instructor dashboard and the CourseStudentFormComponent which were both unused.

This will probably have to be updated after #81 is merged.

### CODE REVIEW

File: www/src/app/controls/course/course-student-form/course-student-form.component.ts [hunk 1]
```diff
@@ -1,310 +0,0 @@
-import {Component, Input, OnInit} from "@angular/core";
```
Comment (wspurgin) [inline www/src/app/controls/course/course-student-form/course-student-form.component.ts:1]: Why is this deleted exactly? Is it just unused?

File: www/src/app/pages/dashboard-page/dashboard-page.component.ts [hunk 2]
```diff
@@ -1,26 +0,0 @@
-import {Component, OnInit} from '@angular/core';
```
Comment (wspurgin) [inline www/src/app/pages/dashboard-page/dashboard-page.component.ts:1]: Same question as above

**Review by wspurgin — COMMENTED** [review]
> Merge remote-tracking branch 'origin/develop' into remove-user-service

We should avoid merging the main branch back into feature/fix branches just to be merged back into the main branch.

File: www/src/app/pages/dashboard-page/dashboard-page.component.ts (hunk 2 above)
Comment (michaeledavis-credera) [reply www/src/app/pages/dashboard-page/dashboard-page.component.ts:1]: Yes, this is the old Dashboard component for instructors. They are now using the admin grid as their dashboard.

File: www/src/app/controls/course/course-student-form/course-student-form.component.ts (hunk 1 above)
Comment (michaeledavis-credera) [reply www/src/app/controls/course/course-student-form/course-student-form.component.ts:1]: Yes. It's unused and wasn't functional anyways.

**Review by wspurgin — COMMENTED** [review]
Once conflicts are resolved, it's good to go.
